diplomacy/web/src/gui/pages/content_game.jsx

```diff
@@ -3646,101 +3656,40 @@ export class ContentGame extends React.Component {
-                                true,
-                                engine,
-                                currentPowerName,
-                                false
-                            )}
-                            {this.renderTabCentaurMessages(
-                                true,
-                                engine,
-                                currentPowerName,
-                                false
-                            )}
-                        </Row>
-                        <Row>
-                            {!engine.isPlayerGame() &&
-                                this.renderPowerInfo(engine)}
-                            {localStorage.getItem("username") === "admin" &&
-                                this.renderLogs(engine, currentPowerName)}
-                        </Row>
-                    </div>
-                );
-            } else {
-                gameContent = (
-                    <div>
-                        <Row>
-                            {phasePanel}
-                            <div className={"col-4"}>{/* Orders. */}</div>
-                        </Row>
-                        <Row>
-                            {this.renderTabChat(
-                                true,
-                                engine,
-                                currentPowerName,
-                                true
-                            )}
-                            {this.renderTabCentaurMessages(
-                                true,
-                                engine,
-                                currentPowerName,
-                                true
-                            )}
-                        </Row>
-                        <Row>
-                            {!engine.isPlayerGame() &&
-                                this.renderPowerInfo(engine)}
-                            {localStorage.getItem("username") === "admin" &&
-                                this.renderLogs(engine, currentPowerName)}
-                        </Row>
-                    </div>
-                );
-            }
-        } else {
-            gameContent = (
-                <div>
-                    {phasePanel}
-                    <Row>
-                        {this.renderTabChat(
-                            true,
-                            engine,
-                            currentPowerName,
-                            true
-                        )}
-                        {this.renderTabCentaurMessages(
+        const receivedSuggestions = advice.filter(
+            (msg) =>
+                msg.type && (msg.type === STRINGS.SUGGESTED_COMMENTARY || msg.type === STRINGS.SUGGESTED_MESSAGE) && (isAdmin || !this.state.annotatedMessages.hasOwnProperty(msg.time_sent))
+        );
+
+        const showMessageAdviceTab = suggestionType !== null && (suggestionType === 1 || suggestionType > 2) && receivedSuggestions.length > 0;
```

Comment:
We should use a more accurate check because `suggestionType > 2` includes some non-message advice types:

```suggestion
        const showMessageAdviceTab = suggestionType !== null && ((suggestionType & UTILS.SuggestionType.MESSAGE) === UTILS.SuggestionType.MESSAGE) && receivedSuggestions.length > 0;
```

Reply:
Shouldn't it also include `UTILS.SuggestionType.COMMENTARY)`?

Reply:
Maybe? Does the variable also control whether the commentary advice tab is shown?

Also, instead of my suggested code, please use the new `hasSuggestionType()` function that I added in #80. That will require rebasing this branch, but you need to resolve conflicts, anyway.

Reply:
I can see you made this suggested change. In the future, please make changes in separate commits from rebases if possible so they are easier to review. If you can't, please at least comment that you made the change as part of the rebase.